README.md

```diff
@@ -0,0 +1,23 @@
+
+
+# Welcome to Sign
+
+## Brief intro to the repo 
+This is a Client Server Application to upload images to server/ S3 bucket
+
+## Getting Started
+Learn how to setup this project and run it on your local machine.
+
+## Dependencies
+This project uses the following dependecies: 
```

Comment:
You could go through the code and find the dependancies. eg: bootstrap, chart library, excel handling library etc.